### ISSUE TITLE
增加 路逕函式  RimeGetSharedDataDir RimeGetUserDataDir RimeGetSyncDir

### ISSUE THREAD

**Issue by shewer** [hchunhui/librime-lua#83]
仿照  LogReg     放入全域   rimeapi  tabel

rimeapi.shared_dir()  -- return string
rimeapi.user_dir()     -- return string
rimeapi.sync_dir()     -- return  string 

```cpp
// types.cc 
namespace RimeapiReg {
        string user_dir(){
                return string(RimeGetUserDataDir());
        };
        string shared_dir(){
                return string(RimeGetSharedDataDir());
        };
        string sync_dir(){
                return string(RimeGetSyncDir());
        };


        static const luaL_Reg funcs[] = {
                {"shared_dir", WRAP(shared_dir)}, // RIME_API const char* RimeGetSharedDataDir();
                {"user_dir",  WRAP(user_dir) }, //RIME_API const char* RimeGetUserDataDir();
                {"sync_dir",  WRAP(sync_dir) },  //RIME_API const char* RimeGetSyncDir();
                { NULL, NULL },
        };
        static const luaL_Reg vars_get[] = {
                { NULL, NULL },
        };
       void init(lua_State *L) {
                lua_createtable(L, 0, 0);
                luaL_setfuncs(L,funcs, 0);
                lua_setglobal(L, "rimeapi");
        };
}

```

**Comment by ZenamLeong** [comment]
If you want to add feature, you should Commit PR.

**Comment by hchunhui** [comment]
merged in #88
